dali/base/dadfs.cpp

```diff
@@ -9567,7 +9571,7 @@ class CDaliDFSServer: public Thread, public CTransactionLogTracker, implements I
         StringArray authScopes;
         CIArrayOf<CFileMatch> matchingFiles;
         start = msTick();
-        count = scanner.getResults(auth, udesc, matchingFiles, authScopes);
+        count = scanner.getResults(auth, udesc, matchingFiles, authScopes, false);
```

Comment:
The OOM problem happens when sorting. As I know, the sorting function will not be called after the iterateFiles(). Am I correct?